### PR TITLE
[TT-11273, DX-1111] Update documentation for 5.0 LTS (dashboard)

### PR DESCRIPTION
**Pull request by buger**
Triggered by: titpetric

Included:

Tyk Gateway: false
Tyk Dashboard: true
Tyk MDCB false
Tyk Pump false

Intended for: 5.0 LTS
Changes sourced from: release-5-lts
Config info generator branch: main

Note: 5.0.11, dashboard only (branch suffix: docs-dashboard-lts)

JIRA: https://tyktech.atlassian.net/browse/TT-11273

### CODE REVIEW

File: tyk-docs/assets/others/dashboard-admin-swagger.yml [hunk 1]
```diff
@@ -1167,6 +1190,146 @@ components:
           x-go-name: ValidateSignature
       type: object
       x-go-package: github.com/TykTechnologies/tyk/apidef
+    ApiDocument:
```
Comment (dcs3spp) [inline tyk-docs/assets/others/dashboard-admin-swagger.yml:1193]: For new schema types ensure that each property has a description. Document examples at the schema level. The example given in schemas appears in the OAS swagger web interface for request and response payloads for all endpoints that use that schema, resulting in better DX. 
Document (where applicable) default value and readonly status of new schema properties.

File: tyk-docs/assets/others/dashboard-admin-swagger.yml [hunk 2]
```diff
@@ -589,19 +589,42 @@ paths:
...
             schema:
-              type: object
+              $ref: '#/components/schemas/OrganisationDocument'
             example:
+              id: 5cc03283d07e7f00019404b3
               owner_name: Jively
+              owner_slug: testorg
               cname: jive.ly
               cname_enabled: true
+              apis:
```
Comment (dcs3spp) [inline tyk-docs/assets/others/dashboard-admin-swagger.yml:607]: Following the statement given in the description, include full example of sub objects, e.g. languages, login_page etc

**Review by dcs3spp — CHANGES_REQUESTED** [review]
Thanks for raising the Dashboard 5 LTS PR separately to facilitate review. Summary of review is as follows:
- Should the version be bumped in dashboard-admin-swagger.yml OAS file also?  
- For each new schema type ensure:
       - each property has a description and, where applicable (default value and readonly status
       - example documented. The example given in schemas appears in the OAS swagger web interface for request and response payloads for all endpoints that use that schema, resulting in better DX.
- Move Organisation example to schema rather than at the endpoint level. Expand the example to include nested type examples, e.g. in ui, event_options and org_options_meta
- Peer review required

**Review by dcs3spp — APPROVED** [review]
@andyo-tyk @titpetric Approved, will raise a ticket relating to requesting changes in this PR

**Review by andyo-tyk — APPROVED** [review]
LGTM

**Pull request merged**